### PR TITLE
br: remove backup and restore principle (#3706)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #3706 to release-3.1

---

### What is changed, added or deleted? (Required)

Remove unnecessary BR principles.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: N/A
- Other reference link(s): N/A

### CODE REVIEW

File: br/backup-and-restore-tool.md [hunk 1]
```diff
@@ -121,13 +65,20 @@ TiKV 收到加载 SST 文件的请求后，利用 Raft 机制保证加载 SST 
...
+<<<<<<< HEAD
 > 聚集这些备份数据可能会造成数据冗余和运维上的麻烦，而且在不聚集这些数据便直接恢复的时候会遇到颇为迷惑的 `SST file not found` 报错。
 > 
 > 更加建议在各个节点挂载 NFS 网盘，或者直接备份到 `S3` 对象存储中。
+=======
+>
+> 聚集这些备份数据可能会造成数据冗余和运维上的麻烦，而且在不聚集这些数据便直接恢复的时候会遇到颇为迷惑的 `SST file not found` 报错。
+>
+> 建议在各个节点挂载 NFS 网盘，或者直接备份到 `S3` 对象存储中。
+>>>>>>> 8cce37e... br: remove backup and restore principle (#3706)
```
Comment (TomShawn) [inline br/backup-and-restore-tool.md:81]: @overvenus Please resolve the conflict.

File: br/backup-and-restore-tool.md [hunk 2]
```diff
@@ -121,13 +65,20 @@ TiKV 收到加载 SST 文件的请求后，利用 Raft 机制保证加载 SST 
 * `"${PDIP}:2379"`：`--pd` 的参数
 
 > **注意：**
-> 
+>
 > 在使用 `local` storage 的时候，备份数据会分散在各个节点的本地文件系统中。
 >
 > **不建议**在生产环境中备份到本地磁盘，因为在日后恢复的时候，**必须**手动聚集这些数据才能完成恢复工作（见[恢复集群数据](#恢复集群数据)）。
+<<<<<<< HEAD
```
Comment (overvenus) [inline br/backup-and-restore-tool.md:72]:
```suggestion
```

File: br/backup-and-restore-tool.md (hunk 1 above)
Comment (overvenus) [inline br/backup-and-restore-tool.md:81]:
```suggestion
> 聚集这些备份数据可能会造成数据冗余和运维上的麻烦，而且在不聚集这些数据便直接恢复的时候会遇到颇为迷惑的 `SST file not found` 报错。
>
> 建议在各个节点挂载 NFS 网盘，或者直接备份到 `S3` 对象存储中。
```

**Pull request merged**